tests/test_session_2_byref_true.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
```

Comment:
filename is too long.  don't make it any longer than `test_session_2.py`

Reply:
Idem.